src/DryIoc/ImTools.cs

```diff
@@ -3806,6 +3806,28 @@ public static ImMapEntry<V> GetEntryOrDefault<V>(this ImMap<V> map, int key)
             return entry != null && entry.Key == key ? entry : null;
         }
 
+        /// <summary>Looks for the sure present entry - in cases when we know for certain that the map contains the entry</summary>
+        [MethodImpl((MethodImplOptions)256)]
+        public static ImMapEntry<V> GetSurePresentEntry<V>(this ImMap<V> map, int key)
```

Comment:
cool, if we know something is presented, than optimize!

Reply:
Yes, my luck that I am in full control of the data structure.